### PR TITLE
zOS: Exploit CEEPCB_3164 indicator bit + propagate CEL4RO31 lookup return code

### PR DESCRIPTION
**Pull request by joransiu**
A few miscellaneous updates and fixes related to CEL4RO31:

- LE added a new CEEPCB_3164 bit on CEEPCBFLAG6 flag within Process
Control Block (PCB) to indicate support for CEL4RO31/64.  Update
omr_cel4ro31_isSupported() and sl_testOpen31bitDLLviaCEL4RO31
to use this new bit.
- Add extern "C" to sltestlib31's function to ensure name is
not C++ mangled.
- Capture the CEL4RO31 return code for function lookup
to the Trc_PRT_sl_lookup_name_Exit2 when function is
not found.

Signed-off-by: Joran Siu <joransiu@ca.ibm.com>

### CODE REVIEW

**Comment by joransiu** [comment]
@babsingh, can you review?  Thanks!

**Comment by joransiu** [comment]
I saw @babsingh might be away.    @tajila / @pshipton, do you mind reviewing?  Thanks!

File: port/zos390/omrsl.c [hunk 1]
```diff
@@ -288,7 +290,7 @@ omrsl_lookup_name(struct OMRPortLibrary *portLibrary, uintptr_t descriptor, char
 	}
 
 	if (address == NULL) {
-		Trc_PRT_sl_lookup_name_Exit2(name, argSignature, handle, 1);
+		Trc_PRT_sl_lookup_name_Exit2(name, argSignature, handle, retcode);
 		return 1;
```
Comment (pshipton) [inline port/zos390/omrsl.c:294]: Should this return retcode? It seems safe to do so.

Comment (joransiu) [reply port/zos390/omrsl.c]: Good idea, and I agree.  Updated under: e15cc29

**Comment by pshipton** [comment]
lgtm @0xdaryl @fjeremic

**Review by pshipton — APPROVED** [review]

**Comment by 0xdaryl** [comment]
Jenkins build all

**Review by 0xdaryl — APPROVED** [review]

**Pull request merged**